<?xml version="1.0" encoding="utf-8"?>
<launch>
	<node pkg="tdmms_finder_teleop" name="tdmms_finder_teleop" type="tdmms_finder_teleop" />
	<node pkg="nic_100a" name="nic_100a_master" type="nic_100a_master"/>
	<node pkg="adm2" name="adm2_master" type="adm2_master"/>
	<node pkg="joy" name="joy_node" type="joy_node"/>
	<node pkg="lv_ncnt_n" name="lv_ncnt_n_master" type="lv_ncnt_n_master" />
        <node pkg="afc_5" name="afc_5_master" type="afc_5_master"/>
        <node pkg="ace2000" name="ace2000_node" type="ace2000_node"/>
        <node pkg="bsc201" name="bsc201_master" type="bsc201_master"/>
</launch>
